geode-redis/src/main/java/org/apache/geode/redis/internal/executor/ExpireAtExecutor.java

```diff
@@ -22,6 +24,7 @@
 import org.apache.geode.redis.internal.ExecutionHandlerContext;
 import org.apache.geode.redis.internal.Extendable;
 import org.apache.geode.redis.internal.RedisConstants.ArityDef;
+import org.apache.geode.redis.internal.RedisDataType;
 import org.apache.geode.redis.internal.RegionProvider;
 
 public class ExpireAtExecutor extends AbstractExecutor implements Extendable {
```

Comment:
The `private final` fields can all be local, except for `ERROR_TIMESTAMP_NOT_USABLE` which is unused

Reply:
I remember killing "ERROR_TIMESTAMP_NOT_USABLE", darn it!